lib/sycamore/sycamore/transforms/table_structure/extract.py

```diff
@@ -174,6 +174,8 @@ def extract(
 
         # Convert the raw objects to our internal table representation. This involves multiple
         # phases of postprocessing.
+        element.properties["backup_tokens"] = tokens
+        element.properties["object"] = objects
```

Comment:
Are these net new properties that we are adding to table elements? Are they something we really want to expose to sycamore users?

Reply:
I know we remove tokens in DocParse. Would we need to remove these props too?

Reply:
oh whoops, those are in from debugging. will drop